database/utils.go

```diff
@@ -18,7 +18,8 @@ func IsKeyNotFoundError(err error) bool {
 
 // Determines the height to process from. There are three cases:
 // 1) The database contains the latest processed block data for the chain
-//   - In this case, we return the maximum of the latest processed block and the configured processHistoricalBlocksFromHeight
+//   - In this case, we return the maximum of the latest processed block and the
+//     configured processHistoricalBlocksFromHeight
```

Comment:
```suggestion
//     configured processHistoricalBlocksFromHeight.
```

nit but you caught the missing period in `config/source_blockchain.go` so might as well here. Although it looks like I'm not in the `CODEOWNERS`file for this repo yet.